jekyll/_cci2/docker-compose.md

```diff
@@ -6,44 +6,56 @@ contentTags:
   platform:
   - Cloud
   - Server v4.x
-  - Server v3.x
 ---
 
-This document describes how to install and use Docker Compose, and assumes the reader has some experience using the `docker-compose` utility.
+This page describes how to use Docker Compose in your CircleCI pipelines.
 
-If you are new to Docker Compose, do consider reviewing the [official Docker Compose overview](https://docs.docker.com/compose/), or checking out the [Getting Started guide](https://docs.docker.com/compose/gettingstarted/).
+If you are new to Docker Compose, you can review the [official Docker Compose overview](https://docs.docker.com/compose/), or checking out the [Getting Started guide](https://docs.docker.com/compose/gettingstarted/).
 
-* TOC
-{:toc}
+## Using Docker Compose with machine executor
+{: #using-docker-compose-with-machine-executor }
+
+If you want to use Docker Compose to manage a multi-container setup with a Docker Compose file, use the `machine` key in your `.circleci/config.yml` file and use `docker compose` as you would normally (see Linux VM execution environment documentation [here]({{site.baseurl}}/using-linuxvm) for more details). That is, if you have a Docker Compose file that shares local directories with a container, this will work as expected.
+
+
+## Using Docker Compose with docker executor
+{: #using-docker-compose-with-docker-executor }
+
+Using the `docker` execution environment combined with `setup_remote_docker` provides a remote engine similar to the one created with running Docker commands and the machine execution environment, but volume mounting and port forwarding do **not** work the same way in this setup. The remote docker daemon runs on a different system than the docker CLI and docker compose, so you must move data around to make this work. Mounting can usually be solved by making content available in a docker volume. It is possible to load data into a Docker volume by using `docker cp` to get the data from the CLI host into a container running on the docker remote host.
```

Comment:
```The remote docker daemon runs on a different system than the docker CLI and docker compose```
This isn't strictly true now. Remote docker jobs now run on a machine executor, and that is where we spin up the jobs containers. The daemon is running on the same executor. I don't know if this can be reworded somehow to leave out such details 😖 Maybe it doesn't matter for your purposes here

Reply:
Yes I wondered about this - so is the whole explanation of moving data aropund to make it accessible outdated now?

This section is there to point out differences (that users need to be aware of) around using docker compose and the docker executor as an alternative to using docker compose and the machine executor. If this info is now irrelevant I can just remove it

Reply:
I suppose it does do that. It is perhaps a bit academic. I see the section has only moved as well. So, I'll give this a stamp